homeassistant/components/switchbot/config_flow.py

```diff
@@ -109,7 +109,7 @@ async def async_step_bluetooth(
             "name": data["modelFriendlyName"],
             "address": short_address(discovery_info.address),
         }
-        if model_name == SwitchbotModel.LOCK:
+        if model_name in {SwitchbotModel.LOCK, SwitchbotModel.LOCK_PRO}:
```

Comment:
Let's make this a constant called `LOCK_MODELS` since its used in two places

Reply:
Added. Called it `SUPPORTED_LOCK_MODELS` so variables in const.py looks more consistent.